src/keymap.cpp

```diff
@@ -139,7 +139,7 @@ static op_desc opdescs[] = {
 	},
 	{
 		OP_SETTAG, "set-tag", "t",
-		_("Select tag"),
+		_("Set tag"),
```

Comment:
Oh alright! I had changed it because otherwise the help dialog showed the same key binding explanation for 't' twice. I'll revert it.